### PR TITLE
Resizing Images

### PR DESCRIPTION
**Pull request by charlesx2013**
Make it so that it doesn't take 10283012 years to load the home page.

@SamLau95

### CODE REVIEW

File: config/locales/application.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Copy this file into application.yml and change env variables as necessary.
+
+SECRET_KEY_BASE: 7f84fa22437fa701a4ecc824af93768fa8764189a0a0c5d9af3de901e0e7e9c4baa4aba6dc86e93ce9955be0a40ecfc4e9649f725a17d01484c75c8fc96cad17
+DEVISE_SECRET_KEY: a97e4eb39c26abb99a9756351156bcfbb916916f9ca7fcc0994b8cbcb24390310e78806e29c052fd1f77ed4c1a6d0285b0440563242ff417acc9fea1e2b7e332
```
Comment (chenhoward) [inline config/locales/application.yml]: WAIT WTF IS THIS

**Comment by chenhoward** [comment]
@charlesx2013 I think you went overboard with the image displayed on the home page.  It's a little blurry on my screen (15 inch retina)

**Comment by chenhoward** [comment]
I'm talking about the Revolv image

**Comment by SamLau95** [comment]
Thanks Charles!

**Pull request merged**